app/models/pageflow/entry_duplicate.rb

```diff
@@ -28,8 +28,8 @@ def copy_draft
     end
 
     def copy_memberships
-      original_entry.users.each do |member|
-        new_entry.memberships.create(user: member)
+      original_entry.memberships.each do |membership|
+        Membership.create(user: membership.user, role: membership.role.to_sym, entity: new_entry)
```

Comment:
I'm not yet convinced always converting roles to symbols is worth it. In cases like this one it mostly adds noise in my opinion. Why are we doing this again?